sdk/index.ts

```diff
@@ -289,6 +291,11 @@ export class Flagsmith {
             } else {
                 this.environment = await request;
             }
+            if (this.environment.identityOverrides?.length) {
+                this.identityOverridesByIdentifier = new Map<string, IdentityModel>(
+                    this.environment.identityOverrides.map(identity => [identity.identifier, identity]
+                    ));
+            }
```

Comment:
I might be missing something here, or perhaps it's just the terminology confusing me slightly, but does this handle the case where a given identity has overrides for multiple features?

Reply:
Then I think what we have here is a map of identifier to identity model, which includes the `identity_features` attribute, right? Should we improve the terminology here (since `identityOverrides` isn't strictly correct), or perhaps add a comment explaining the behaviour? I think it's a little unclear what's happening.

Reply:
Changed to `this.identitiesWithOverridesByIdentifier`.

Reply:
Thanks, I think that's better. I was trying to work out if there was an improvement we could also make to the `identityOverrides` attribute on the EnvironmentModel class but I'm not sure if it's worthwhile. 

Arguably, given this change it could be `identitiesWithOverrides`, but probably this is overkill. What do you think?

Reply:
As discussed elsewhere, we're staying with `EnvironmentModel.identityOverrides` to avoid extra renames in Edge API + other engine implementations.